ferrocene/doc/user-manual/src/rustfmt/install.rst

```diff
@@ -0,0 +1,37 @@
+.. SPDX-License-Identifier: MIT OR Apache-2.0
```

Comment:
Thinking about it, I'd prefer if we just link to the main installation page (https://public-docs.ferrocene.dev/main/user-manual/install.html). That page is more accurate, and will be expanded as we add more installation methods (like criticalup).

That page says:

> For each of the targets you identified above, download all the archives listed in the “installation archives” section of the target documentation.

So we should add a "The following archives are needed when installing the optional rustfmt tool:" in https://public-docs.ferrocene.dev/main/user-manual/targets/x86_64-unknown-linux-gnu.html#archives-to-install